tldw_Server_API/app/core/DB_Management/Resource_Daily_Ledger.py

```
@@ -0,0 +1,218 @@
+                    entry.category,
+                    int(entry.units),
+                    entry.op_id,
+                    entry.occurred_at.isoformat(),
+                )
+                # Prefer rowcount when available: 1 if inserted, 0 if ignored
+                try:
+                    rc = int(getattr(res, "rowcount", -1))
+                    if rc in (0, 1):
+                        return rc == 1
+                except Exception:
+                    pass
```

Ruff (0.14.3)

`try`-`except`-`pass` detected, consider logging the exception

(S110)